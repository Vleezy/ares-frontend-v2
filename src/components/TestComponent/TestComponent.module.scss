.testComponent {
    margin-top: 0;
}
