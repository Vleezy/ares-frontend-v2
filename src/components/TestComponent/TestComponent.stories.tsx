import { Story } from "@storybook/react/types-6-0";
import React, { ComponentProps } from "react";
import {
    TestComponent as TestComponentComponent,
    Props,
} from "./TestComponent";

// This default export determines where your story goes in the story list
const Component = {
    title: "Components/TestComponent",
    component: TestComponentComponent,
    argTypes: {
        onClick: { action: "clicked" },
    },
};

const Template: Story<ComponentProps<typeof TestComponentComponent>> = (
    args
) => <TestComponentComponent {...args} />;

export const TestComponent = Template.bind({});
export default Component;

TestComponent.args = {
    group: "admin",
} as Props;
