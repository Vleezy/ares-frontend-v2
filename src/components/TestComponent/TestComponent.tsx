import s from "./TestComponent.module.scss";

import React, { FC } from "react";

export type Props = {};

export const TestComponent: FC<Props> = ({}) => {
    return <div className="test-component">TestComponent Works!</div>;
};
