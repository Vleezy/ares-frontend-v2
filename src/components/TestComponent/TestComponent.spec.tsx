import React from "react";

import { render } from "@testing-library/react";

import { TestComponent } from "./TestComponent";

describe("Test Component Component", () => {
    it("should render properly", () => {
        const { container } = render(<TestComponent />);

        expect(container.firstChild).toMatchSnapshot();
    });
});
