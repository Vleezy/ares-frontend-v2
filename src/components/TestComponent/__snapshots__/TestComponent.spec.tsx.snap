// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Component Component should render properly 1`] = `
<div
  class="test-component"
>
  TestComponent Works!
</div>
`;
